Userland/Libraries/LibAudio/FlacTypes.h

```diff
@@ -84,13 +86,38 @@ struct FlacRawMetadataBlock {
     FlacMetadataBlockType type;
     u32 length; // 24 bits
     ByteBuffer data;
+
+    ErrorOr<void> write_to_stream(Stream&) const;
 };
 
 enum class BlockingStrategy : u8 {
     Fixed = 0,
     Variable = 1,
 };
 
+// Block sample count can be stored in one of 5 ways.
+enum class BlockSizeCategory : u8 {
+    Reserved = 0b0000,
+    S192 = 0b0001,
+    // The formula for these three is 144 * (2^x), and it appears to be an MP3 compatibility feature.
```

Comment:
Three?

Reply:
The three following block sizes

Reply:
The fourth following block size is there without a comment then?